Minio.Functional.Tests/Program.cs

```diff
@@ -50,14 +50,7 @@ public static void Main(string[] args)
         }
 
         MinioClient minioClient = null;
-
-        var clientHandler = new HttpClientHandler();
-        clientHandler.ServerCertificateCustomValidationCallback = (sender, cert, chain, sslPolicyErrors) =>
-        {
-            return true;
-        };
-        clientHandler.UseProxy = false;
-        var httpClient = new HttpClient(clientHandler);
```

Comment:
did you meant to remove this? @ebozduman

Reply:
No.This change is supposed to be part of the PR #641.
Thank you for catching it.